wiki/Game_modifier/Spun_Out/zh-tw.md

```diff
@@ -0,0 +1,33 @@
+---
+stub: true
+tags:
+  - spun-out
+  - spun out
+  - SO
+  - mod
+  - game modifier
+  - 遊戲 Mod
+---
+
+# Spun Out (Mod)
+
+![SO Mod 圖示](/wiki/shared/mods/SO.png "Spun Out (SO) Mod 圖示")
+
+*若要查看 Mod 列表，請見：[遊戲 Mod](/wiki/Game_modifier)*
+
+## 關於
+
+- 別名：SO
+- 類型：特殊
+- 成績加乘：0.90 倍
+- 預設快捷鍵：`C`
+- 註解：`轉盤將自動完成。`
+- 適用的遊戲模式：![][osu!]
+
+## 說明
+
+**Spun Out** Mod 能夠輔助 [osu!](/wiki/Game_mode/osu!) 玩家在遊玩中轉轉盤。Mod 將自動以每分鐘 286.48 轉 (SPM) 的速度 (順時針) 自動轉完轉盤。恰好是 Auto Mod 轉轉盤速度 (每分鐘 477.26 轉) 的五分之三。
```

Comment:
missing link to `/wiki/Beatmaps/`

Reply:
I skipped the beatmap because translate it would make the sentence looks weird